web/book/src/project/integrations/postgresql.md

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL
+
+PL/PRQL is a PostgreSQL extension that lets you write functions with PRQL. For example:
+
+```sql
+create function player_stats(int) returns table(player text, kd_ratio real) as $$
+  from rounds
+  filter match_id == $1
+  group player (
+    aggregate {
+      total_kills = sum kills,
+      total_deaths = sum deaths
+    }
+  )
+  filter total_deaths > 0
+  derive kd_ratio = total_kills / total_deaths
+  select { player, kd_ratio }
+$$ language plprql
+
+```
+
+You can also pass PRQL code to the `prql` function. For example:
+
+ ```
+ select prql('from rounds | filter match_id == 1, 'rounds_cursor');
+ ```
+
+For more information on the extension, see the [PL/PRQL repository](https://github.com/kaspermarstal/plprql).
+
+# Getting started
+On Ubuntu, follow these steps to install PL/PRQL from source:
+
+1. Install `cargo-pgrx`.
+
+    ```cmd
```

Comment:
```suggestion
    ```sh
```

Reply:
Thanks, but we'll link the README in the repo and delete the code block from here. Will make sure to incorporate your suggestions there.